src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -138,7 +135,7 @@
     /**
      * Update the summary statistics.
      */
-    static final boolean UPDATE_STATISTICS = readBooleanPropertyPrefixed("statistics", true);
+    static final boolean UPDATE_STATISTICS = readBooleanPropertyPrefixed("statistics", false);
```

Comment:
Was the UPDATE_STATISTICS default value change intentional? I'm supportive of disabling most of these, but I've found `getActiveCount()` is a helpful tool to monitor load. Thoughts on using a separate property for a subset of these?

Reply:
Yes, they have a pretty substantial performance overhead.  I have a note to see if this method can be implemented by returning the size of the active threads set though...